src/app/workloads/workloads.component.ts

```diff
@@ -279,6 +281,7 @@ export class WorkloadsComponent implements OnInit, OnDestroy {
       .getPodDetails(this.currentRoute[1], this.currentRoute[1])
       .subscribe(res => {
         this.litmuspod = res.statefulSet;
+        this.mayapvc = res.pvc.pvc
```

Comment:
Why this two times pvc used?

Reply:
pvc is stored inside the pvc
something like this
```
pvc {
  numberofPvc: number;
  pvc: 
    {
      name: string;
      volumeName: string;
    }
}
```

Reply:
change the pcv -> pvcs { pvc {} }